2750. Ways to Split Array Into Good Subarrays/solution.py

```diff
@@ -1,11 +1,26 @@
 class Solution:
     def numberOfGoodSubarraySplits(self, nums: List[int]) -> int:
         # Iterate until the first 1. 
+        i = 0
+        try:
```

Comment:
Note: This doesn't work without the try-except block if the test case is [0, 0].